app/templates/components/book-cover.hbs

```diff
@@ -1,7 +1,11 @@
-<li {{action "open"}}>
-  <strong>{{book.title}}</strong><br><br>
+<li >
+  <strong{{action "open"}}>{{book.title}}</strong><br><br>
```

Comment:
Sounds good to add `role="button"` here. I've never heard of that before, but a quick googling showed it's importance!